akka-http/src/main/scala/akka/http/scaladsl/server/directives/FormFieldDirectives.scala

```diff
@@ -40,6 +59,50 @@ trait FormFieldDirectives extends ToNameReceptacleEnhancements {
 }
 
 object FormFieldDirectives extends FormFieldDirectives {
+
+  val _formFieldSeq: Directive1[immutable.Seq[(String, String)]] = {
+    import BasicDirectives._
+    import FutureDirectives._
+    import akka.http.scaladsl.unmarshalling._
+
+    extract { ctx ⇒
+      import ctx.{ executionContext, materializer }
+      Unmarshal(ctx.request.entity).to[StrictForm].fast.flatMap { form ⇒
+        val fields = form.fields.collect {
+          case (name, field) if name.nonEmpty ⇒
+            Unmarshal(field).to[String].map(fieldString ⇒ (name, fieldString))
```

Comment:
Now that I see it, this directive implies we'll try to toStrict all parts of the FormData...
I'd like to avoid attempting this for all fields - since one of them may be a "very big thing" (i.e. file upload, in the "file" form field). Making it `formFieldSeq("bla", "bla2")` makes it more explicit, but doesn't really solve the problem. I currently don't have an idea how to address my above worry – should we don't worry about it but document these semantics explicitly, or not provide such directive - WDYT @sirthias @jrudolph ?